publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/tickets/DoiRequestEventProducer.java

```diff
@@ -75,11 +75,16 @@ protected DoiMetadataUpdateEvent processInputPayload(
                    ? propagateEvent(eventBody)
                    : EMPTY_EVENT;
     }
-    
+
+    private static String readDomainName() {
```

Comment:
Personally, I don't like these kind of lookups in the environment. It is hard to see what environment variables a given handler depends on. I would have looked up the domain name in the environment in the default constructor of the handler. But I know the publication api is full of these kind of lookups in the environment, so it is ok to leave it as is too.